src/main/java/ch/njol/skript/expressions/ExprExcept.java

```diff
@@ -0,0 +1,81 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.LiteralUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Except")
+@Description("Filter a list by providing objects to be excluded.")
+@Examples({
+	"spawn zombie at location(0, 0, 0):",
+		"\thide entity from all players except {_player}",
+	"",
+	"set {_items::*} to a copper ingot, an iron ingot and a gold ingot",
+	"set {_except::*} to {_items::*} excluding copper ingot"
+})
+@Since("INSERT VERSION")
+public class ExprExcept extends SimpleExpression<Object> {
+
+	static {
+		Skript.registerExpression(ExprExcept.class, Object.class, ExpressionType.COMBINED,
+			"%objects% (except|excluding|not including) %objects%");
```

Comment:
```suggestion
			"%~objects% (except|excluding|not including) %objects%");
```
There is no need to allow literals here